react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,32 @@
+i {
+  font-size: 2rem;
+}
+
+.menu {
+  margin-top: 5px;
+}
+
+p,
+.menu {
+  margin-left: 15px;
+}
+
+p:hover,
+.menu:hover,
+i:hover {
+  cursor: pointer;
+}
+
+.row {
+  width: 250px;
+  height: 100vh;
+  border-right: 2px solid black;
+  position: absolute;
+  bottom: 1px;
+  left: -1px;
+  box-shadow: 0 0 0 1050px rgba(0, 0, 0, 0.72);
```

Comment:
The "shade" should really be its own element / click target. Try adding an additional div. You'll likely need to apply fixed/absolute positioning to it